applications/nrf_desktop/src/hw_interface/leds.c

```diff
@@ -76,6 +127,9 @@ static void work_handler(struct k_work *work)
 			if (led->effect->loop_forever) {
 				led->effect_step = 0;
 			}
+			if (led->stream) {
```

Comment:
Assert that streaming does not have `loop_forever` set? Otherwise stream may get blocked here.

Reply:
loop_forever have no effect on led_stream because led_stream always set led->effect_step to 0